public/javascripts/SVLabel/src/SVLabel/Storage.js

```diff
@@ -39,6 +39,10 @@ function TemporaryStorage(JSON, params) {
             set("completedOnboarding", null);
         }
 
+        if (!get("completedMissionAnonymously")){
```

Comment:
Right, I checked that it is done. How do is done currently? Do we make use of the `completedFirstMission` function or is this function used only for anonymous users?